src/ApacheOrcDotNet.OptimizedReader/Buffers/BinaryColumnBuffer.cs

```diff
@@ -0,0 +1,91 @@
+﻿using ApacheOrcDotNet.OptimizedReader.Infrastructure;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+
+namespace ApacheOrcDotNet.OptimizedReader.Buffers
+{
+    [SkipLocalsInit]
```

Comment:
Removed since we reduced the usage of `stackalloc` and the places where it is still used, the buffers are very small.

One good post I found about this is [here](https://blog.ladeak.net/posts/skipinitlocal).

Reply:
That post is a little weird...  First that it takes 30μS to set 8192 bytes to 1...  Then that there's only a negligible difference between setting them to zero first and then setting them to 1.

Anyway, if we do believe that post then I wouldn't use `[SkipLocalsInit]`, as for reasonably-sized buffers it's actually faster to zero-init for some reason, and for large buffers the delta is within the standard deviation.  Not worth the safety loss in my opinion.